sdl-core/src/main/scala/io/smartdatalake/workflow/action/ActionHelper.scala

```diff
@@ -127,6 +127,7 @@ private[smartdatalake] object ActionHelper extends SmartDataLakeLogger {
     invalidCharacters.replaceAllIn(str, "_")
   }
 
+  // TODO: remove
```

Comment:
Are these deprecated then? Or can they be removed now?

Reply:
yes, can be removed, thanks!